compile.sh

```diff
@@ -180,13 +179,13 @@ echo
 
 # Echo: Compilation complete.
 popd
-pushd MCServer/MCServer
+pushd cuberite/cuberite
 echo
 echo "-----------------"
 echo "Compilation done!"
 echo
 echo "Cuberite awaits you at:"
-echo "`pwd`/MCServer"
+echo "`pwd`/Cuberite"
```

Comment:
If the output executable name is still MCServer, then this is indeed wrong (until the output executable name is updated).